pywbem/cim_operations.py

```diff
@@ -1709,6 +1741,299 @@ def OpenEnumerateInstancePaths(self, ClassName, namespace=None,
+              :attr:`~pywbem.CIM_ERR_CONTINUATION_ON_ERROR_NOT_SUPPORTED`.
+            * If `False`, the server is requested to close the enumeration after
+              sending an error response.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default behaviour to be used.
+              :term:`DSP0200` defines that the server-implemented default is
+              `False`.
+
+            Many WBEM servers do not support this request attribute so that
+            its use is NOT recommended except in special cases.
+
+            If this attribute is `True` and the method uses the
+            EnumerateInstances option (`use_pull_operations` == `False`
+            or the server does not support pull operations), the method will
+            fail with a ValueError
+
+          MaxObjectCount (:class:`~pywbem.Uint32`)
+            Maximum number of instances the WBEM server may return
+            for each of the open and pull requests issues as part of the
+            execution of this generator.
+
+            * If positive, the WBEM server is to return no more than the
+              specified number of instances.
+            * Zero is not allowed in that it would mean that zero instances
+              are to be returned for every request issued.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default behaviour to be used.
+              :term:`DSP0200` defines that the server-implemented default is
+              to return zero instances.
+
+          The choice of MaxObjectCount is client/server dependent but choices
+          between 100 and 1000 typically do not have a significant impact on
+          either memory or overall efficiency.
+
+        Keyword Arguments:
+
+          extra :
+            Additional keyword arguments are passed as additional operation
+            parameters to the WBEM server.
+            Note that :term:`DSP0200` does not define any additional parameters
+            for this operation.
+
+        Raises:
+
+            Exceptions described in :class:`~pywbem.WBEMConnection`.
+
+        Returns:
+
+          A Python iterator object. Instances can be retrieved
+          by iterating through the iterator.
+
+        Example:
+            insts_iterator = conn.IterEnumerateInstances('CIM_Blah')
+            for inst in insts_iterator:
+                # close if a particular property value found
+                if inst.get('thisPropertyName') == 0
+                    enum_get.close()
+                    break;
+                else:
+                    print('instance %s' % inst.tomof())
+
+        """
+
+
+        if MaxObjectCount is not None and MaxObjectCount == 0:
+            raise ValueError('MaxObjectCount=%s. Invalid leq 0' % \
+                             MaxObjectCount)
```

Comment:
The message still does not reflect the comparison. In addition, we should not use abbreviations such as `leq`.